frontend/src/components/student/AssessmentExperience.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react";
+import { Box, HStack, Spacer, Text, VStack } from "@chakra-ui/react";
+
+import type { TestResponse } from "../../APIClients/types/TestClientTypes";
+import AssessmentExperienceContext from "../../contexts/AssessmentExperienceContext";
+import type { Answers } from "../../types/AnswerTypes";
+import { initializeAnswers } from "../../utils/StudentUtils";
+import useReloadPrompt from "../common/navigation/useReloadPrompt";
+
+import Question from "./questions/Question";
+import QuestionTitle from "./questions/QuestionTitle";
+import Instructions from "./Instructions";
+import NavButtons from "./NavButtons";
+import QuestionNumbers from "./QuestionNumbers";
+import StudentDashboardHeader from "./StudentDashboardHeader";
+
+interface AssessmentExperienceProps {
+  test: TestResponse | null;
+  className: string;
+}
+
+const AssessmentExperience = ({
+  test,
+  className,
+}: AssessmentExperienceProps): React.ReactElement => {
+  useReloadPrompt();
```

Comment:
Is it intended to have this prompt both here and in `WriteAssessmentPage.tsx`? I imagine that might result in some wonky behaviour... I imagine you could remove it here since we don't need it when previewing?

Reply:
oops, meant to remove here :D good catch - thank you!